pkg/asset/manifests/kubevirt/cloudproviderconfig.go

```diff
@@ -0,0 +1,34 @@
+package kubevirt
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// CloudProviderConfig is the kubevirt cloud provider config
+type CloudProviderConfig struct {
```

Comment:
we don't have a cloud provider. is this being used somewhere?

Reply:
Yes, it used to pass the namespace and the infraID to cluster-api-provider-kubevirt